### PR TITLE
Use verifyBracketsWellFormed in complete provision script

### LINKED ISSUE #218: Verify brackets are well formed in complete provision
Replace this chunk of code which asserts only that masterSafe is the sole owner of the provided brackets:

https://github.com/gnosis/dex-liquidity-provision/blob/00398b3ef932d1478a1cc1a9f5865d115970986c/scripts/complete_liquidity_provision.js#L123-L129

with a more strict verification regarding many more strict details about the brackets provided:

https://github.com/gnosis/dex-liquidity-provision/blob/00398b3ef932d1478a1cc1a9f5865d115970986c/scripts/utils/verify_scripts.js#L20-L26

Depends on #217

### PR DESCRIPTION
**Pull request by bh2smith**
Introducing the use of `verifyBracketsWellFormed` which asserts on many aspects of the provided bracket safes so to prevent any malicious activity. _Some_ facts regarding the bracket safes that are ensured here are

- that the owner of the brackets is the masterSafe
- that masterCopy of brackets is the known masterCopy
- proxy bytecode
- absence of modules
- unchanged fallback handler

If any of the above verification descriptions seem incomplete, please say so and suggest some improvements.

Closes #218 

### Test Plan:

```sh
git checkout 218/use_verifyBracketsWellFormed
export PK=0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d
npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=1 --quoteTokenId=4 --lowestLimit=150 --highestLimit=200 --currentPrice=175 --masterSafe=0x8990c564ec303C7b26d3d5556ef0910E58Be08Ce --depositBaseToken=0.01 --depositQuoteToken=1 --fleetSize=2 --network=rinkeby --verify --nonce=2 --brackets=0xc030CBAdcb3933Bc64D59f893eEb8990C96A7677,0xc7b50c03f2e91BF8F5D1A83A3D7A5F471D6b4229
```

### Additional Notes

These changes were temporarily blocked by #217 which is yet to be solved. However, we have temporarily lifted of one of the verifies in order to push this along. The temporarily ignored verification is that the bytecode of the provided brackets are indeed in correspondence with a trusted GnosisSafeProxy contract.

Perhaps we could warn the user that this verification is being skipped so that at least they are aware? @josojo What do you think about this?

### CODE REVIEW

File: test/verification_script.js [hunk 1]
```diff
@@ -127,12 +127,12 @@ contract("Verification checks", function (accounts) {
     await exchange.placeOrder(targetToken, stableToken, 1234124, 11241234, 11234234, { from: accounts[0] })
   })
   describe("Owner is master safe", async () => {
-    it("throws if the masterSafe is not the only owner", async () => {
+    it.only("throws if the masterSafe is not the only owner", async () => {
```
Comment (bh2smith) [inline test/verification_script.js:130]:
```suggestion
    it("throws if the masterSafe is not the only owner", async () => {
```

File: scripts/utils/sign_and_send.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     }
     const safeTxGas = await estimateGas(masterSafe, transaction)
     const baseGas = 0
-    console.log("Aquiring Transaction Hash")
+    console.log("Acquiring Transaction Hash")
```
Comment (josojo) [inline scripts/utils/sign_and_send.js:44]: Can we delete this log? Its not useful at all

Comment (bh2smith) [reply scripts/utils/sign_and_send.js:44]: Sure thing
```suggestion
```

**Comment by josojo** [comment]
Why is the PR only in draft? It looks good already.

**Comment by bh2smith** [comment]
> Why is the PR only in draft? It looks good already.

It is still failing because of #217 which I plan to also fix in this PR. Please help!

**Pull request merged**